### PR TITLE
Adds a new blog post for partial dumps

### PR DESCRIPTION
**Pull request by kiwicopple**
https://zone-www-dot-19vzigdxd-supabase.vercel.app/blog/2022/06/28/partial-postgresql-data-dumps-with-rls

### CODE REVIEW

File: apps/www/public/rss.xml [hunk 1]
```diff
@@ -5,791 +5,799 @@
...
+  <pubDate>Thu, 16 Jun 2022 05:00:00 GMT</pubDate>
 </item>
 
 <item>
   <guid>https://supabase.com/blog/2022/06/07/beta-update-may-2022</guid>
   <title>Supabase Beta May 2022</title>
   <link>https://supabase.com/blog/2022/06/07/beta-update-may-2022</link>
   <description>Product and community updates including wildcard auth redirects, edge functions with webhooks, and Prometheus endpoints for everybody.</description>
-  <pubDate>Wed, 01 Jun 2022 00:00:00 GMT</pubDate>
+  <pubDate>Tue, 31 May 2022 05:00:00 GMT</pubDate>
```
Comment (MildTomato) [inline apps/www/public/rss.xml:32]: we need to fix this rss feed generator. it keeps using the timezone that the local machine is in

File: apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
+---
+title: Partial data dumps using Postgres Row Level Security
+description: Using RLS to dump only a subset of your database.
```
Comment (kiwicopple) [inline apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx:3]:
```suggestion
description: Using RLS to create seed files for local PostgreSQL testing.
```

**Review by olirice — APPROVED** [review]

**Review by steve-chavez — APPROVED** [review]
Looks really practical and useful :+1:

File: apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  for select
+  to exporter
+  using ( is_exportable = true );
+
+```
+
+## Conclusion
+
+Using `seed` data isn't the only way to run development environments. It's also possible to run fully-masked copies of your database using tools like
+[Snaplet](https://docs.snaplet.dev/tutorials/supabase-clone-environments).
```
Comment (steve-chavez) [inline apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx:172]: This link is not working for me: https://docs.snaplet.dev/tutorials/supabase-clone-environments

Comment (steve-chavez) [reply apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx:174]: Unfortunately it doesn't:

Screenshot from 2022-06-30 16-04-37

Comment (kiwicopple) [reply apps/www/_blog/2022-06-28-partial-postgresql-data-dumps-with-rls.mdx:174]: I'll ping the snaplet team

**Pull request merged**